### PR TITLE
Metadata ordering

### PR DESCRIPTION
**Pull request by hbs**
Internal modifications so classes are scanned in the order of their classId, thus returning results in lexicographic order of classId+labelsId.

### CODE REVIEW

File: warp10/src/main/java/io/warp10/continuum/DirectoryUtil.java [hunk 1]
```diff
@@ -126,7 +126,7 @@ private static long computeHash(long k0, long k1, long timestamp, List<String> c
   public static DirectoryStatsResponse stats(DirectoryStatsRequest request,
                                              StandaloneDirectoryClient.ShardFilter filter,
                                              Map<String, Map<Long, Metadata>> metadatas,
-                                             Map<String, Set<String>> classesPerOwner,
+                                             Map<String, Map<Long, String>> classesPerOwner,
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/continuum/DirectoryUtil.java:129]: Update copyright notice.

File: warp10/src/main/java/io/warp10/standalone/StandaloneDirectoryClient.java [hunk 2]
```diff
@@ -43,6 +45,8 @@
 import java.util.regex.Pattern;
 
 import io.warp10.continuum.DirectoryUtil;
+import io.warp10.continuum.MetadataUtils;
+
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/standalone/StandaloneDirectoryClient.java:49]: Unused.

File: warp10/src/main/java/io/warp10/standalone/StandaloneDirectoryClient.java [hunk 3]
```diff
@@ -67,6 +71,7 @@
 import io.warp10.continuum.Configuration;
 import io.warp10.continuum.egress.ThriftDirectoryClient;
 import io.warp10.continuum.gts.GTSHelper;
+import io.warp10.continuum.gts.MetadataIdComparator;
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/standalone/StandaloneDirectoryClient.java:74]: Unused.

File: warp10/src/main/java/io/warp10/standalone/StandaloneDirectoryClient.java [hunk 4]
```diff
@@ -110,11 +115,27 @@
   private long LIMIT_CLASS_CARDINALITY = 100;
   private long LIMIT_LABELS_CARDINALITY = 100;
   
+  private static final int CACHE_SIZE = 1000000;
+
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/standalone/StandaloneDirectoryClient.java:119]: Unused.

**Review by ftence — CHANGES_REQUESTED** [review]

**Comment by hbs** [comment]
PTAL

**Review by ftence — APPROVED** [review]

**Pull request merged**